docs/readme.md

```diff
@@ -78,6 +78,8 @@ Release Notes
       as a Java file instead of precompiling it. This is to better support
       changes with the UnityPlayerActivity, and GameActivity options, in
       the Unity 2023 editor.
+    - Firestore: Added `Count()`, which fetches the number of documents in
```

Comment:
Add `([#659](https://github.com/firebase/firebase-unity-sdk/pull/659))` to the release notes entry, so customers can easily correlate this change log entry with the corresponding code change.